specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromIds.tsx

```diff
@@ -183,38 +183,44 @@ export function RecordSelectorFromIds<SCHEMA extends AnySchema>({
       <ResourceView
         dialog={dialog}
         headerButtons={(specifyNetworkBadge): JSX.Element => (
-          <>
-            {headerButtons}
-            <DataEntry.Visit
-              resource={!isDependent && dialog !== false ? resource : undefined}
-            />
-            {hasTablePermission(model.name, isDependent ? 'create' : 'read') &&
-            typeof handleAdding === 'function' ? (
-              <DataEntry.Add
-                aria-label={addLabel}
-                disabled={mode === 'view'}
-                title={addLabel}
-                onClick={handleAdding}
+          <div className="flex flex-col items-center gap-2 md:flex-row md:gap-8 ">
+            <div className="flex gap-2" id="headerButtons">
+              {headerButtons}
+              <DataEntry.Visit
+                resource={
+                  !isDependent && dialog !== false ? resource : undefined
+                }
               />
-            ) : undefined}
-            {typeof handleRemove === 'function' && canRemove ? (
-              <DataEntry.Remove
-                aria-label={removeLabel}
-                disabled={resource === undefined || mode === 'view'}
-                title={removeLabel}
-                onClick={(): void => handleRemove('minusButton')}
-              />
-            ) : undefined}
-            {typeof newResource === 'object' ? (
-              <p className="flex-1">{formsText.creatingNewRecord()}</p>
-            ) : (
-              <span
-                className={`flex-1 ${dialog === false ? '-ml-2' : '-ml-4'}`}
-              />
-            )}
-            {specifyNetworkBadge}
-            {slider}
-          </>
+              {hasTablePermission(
+                model.name,
+                isDependent ? 'create' : 'read'
+              ) && typeof handleAdding === 'function' ? (
+                <DataEntry.Add
+                  aria-label={addLabel}
+                  disabled={mode === 'view'}
+                  title={addLabel}
+                  onClick={handleAdding}
+                />
+              ) : undefined}
+              {typeof handleRemove === 'function' && canRemove ? (
+                <DataEntry.Remove
+                  aria-label={removeLabel}
+                  disabled={resource === undefined || mode === 'view'}
+                  title={removeLabel}
+                  onClick={(): void => handleRemove('minusButton')}
+                />
+              ) : undefined}
+              {typeof newResource === 'object' ? (
+                <p className="flex-1">{formsText.creatingNewRecord()}</p>
+              ) : (
+                <span
+                  className={`flex-1 ${dialog === false ? '-ml-2' : '-ml-4'}`}
+                />
+              )}
+              {specifyNetworkBadge}
+            </div>
+            <div id="slider">{slider}</div>
```

Comment:
Why was ID added?

Also. Note that IDs must be unique on the page. If this component get's rendered more than once on the page, you have a bug. If you really need an id (and you don't here), you should use the `useId` hook which will make sure to give you a unique id